packages/ts-codegen/src/__tests__/index.test.ts

```diff
@@ -744,6 +744,310 @@ describe(generateTS, () => {
     });
   });
 
+  it("import an entity, which is used to implement interface and returned by resolver", () => {
+    let { resolvers, models } = runGenerateTest(graphql`
+      extend schema @import(from: "@msteams/packages-test", defs: ["Entity"])
```

Comment:
Hmmm, did we agree on shortening `definitions` in a spec? I'm not really a fan.

Reply:
@freiksenet what do you think? I am fine with both.

Reply:
This doesn't need to block this PR btw.